docs/runbooks/deploy-agent-cloud-run.md

```
@@ -0,0 +1,111 @@
+gcloud iam workload-identity-pools providers create-oidc "$PROVIDER" \
+  --project="$PROJECT_ID" --location=global \
+  --workload-identity-pool="$POOL" \
+  --display-name="GitHub" \
+  --attribute-mapping="google.subject=assertion.sub,attribute.repository=assertion.repository" \
+  --attribute-condition="assertion.repository=='${REPO}'" \
+  --issuer-uri="https://token.actions.githubusercontent.com"
+```
+
+Record the provider resource name:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)